### ISSUE TITLE
bindable without view-model

### ISSUE THREAD

**Issue by mikefills** [aurelia/framework#357]
is there any reason why using `bindable` in the template only works without view-model?
myComponent.html file:
```
<template bindable='h1,h2'>
  ${h1} - ${h2}
</template>
```
it works only if you use <require from="./expander.html"></require>
I think this is a very cool feature so why doesn't it also work when you have a view-model?
(it would also be nice to update documentation to let people know you can actually use as several comma delimited parameters not just one)

**Comment by Thanood** [comment]
I don't know if it is the intention but from my point of view it can become confusing if you were able to declare bindables in both view and view model. It's clearer to have these things in one place, IMO.

**Comment by mikefills** [comment]
I agree with you, but think of the `ref` attribute for example, it does that already. 
Second, you wouldn't have to use it if you don't want to. It would spare some lines from having to declare `@bindable` in the view-model.

**Comment by atsu85** [comment]
I'd vote for NOT changing current implementation, for the same reasons @Thanood gave:
> I don't know if it is the intention but from my point of view it can become confusing if you were able to declare bindables in both view and view model. It's clearer to have these things in one place, IMO.

I understand @mikefills point:
> Second, you wouldn't have to use it if you don't want to.
My point is it would be just an option having to declare @bindable in the view-model.

but the down side is, that every time You want to use a custom element, You'll need to check 2 files instead of one.
